localization/english/dynasties/wc_dynasty_names_l_english.yml

```diff
@@ -429,6 +429,10 @@
  dynn_Atlar:0 "Atlar"
  dynn_Lapidis:0 "Lapidis"
  dynn_Bandor:0 "Bandor"
+
+ # Easter Eggs
+ dynn_Greenwood:0 "Greenwood"
+ dynn_Hasic:0 "of Hasic"
```

Comment:
New localization lines must be at the end of the file.

Reply:
Fixed in 7b2b72e109bcdea81d141edeb6ad82bdba68f8e4.